frontend/src/components/SignInForm.js

```diff
@@ -0,0 +1,57 @@
+import React, {useState} from 'react' ;
+
+export const SignInForm = () => {
+
+  const [errorMessage, setErrorMessage] = useState('')
+  const [email, setEmail] = useState('')
+  const [password, setPassword] = useState('')
+
+  const onLoggedIn = (event) => {
+    console.log('login successfull')
+    window.location.href='/Secrets'
+ }

```

Comment:
So, here you are actually changing the web address of the browser, rather than telling React which component to load.

From this example: https://stackoverflow.com/questions/31079081/programmatically-navigate-using-react-router

You can use 
```javascript
import { useHistory } from "react-router-dom";
...
const onLoggedIn = (event) => {
    console.log('login successfull')
    history.push("/Secrets");
 }
```

This code tells react where to go based on the Routes and Components you've defined